src/app/public/modules/file-attachment/file-item-error-type.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Specifies the type of error that was thrown.
+ */
+export type SkyFileItemErrorType = 'fileType' | 'minFileSize' | 'maxFileSize' | 'validate';
```

Comment:
If consumers need this type, shouldn't we be exporting it in the `public_api.ts` file?